DemoServer/AdditionalLanguages/python/staticIndexes/fanoutIndex/metadata.json

```diff
@@ -0,0 +1,119 @@
+{
+    "Slug": "fanout-index",
+    "SourceFileName": "fanoutIndex.py",
+    "Title": "Fanout Index",
+    "DescriptionLines": [
+        "A **Fanout Index** creates multiple index entries per processed document.",
+        "",
+        "For example, when a document field is an array, an index entry can be generated per array element,",
+        "resulting in multiple index entries for the document.",
+        "",
+        "Since this can be an expensive operation, a performance hint is issued in the Studio when the fanout ratio is high (number is configurable)."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Static Indexes Overview Demo",
+            "Type" : "demo",
+            "Url"  : "/python/static-indexes/static-indexes-overview"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the Map index class",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Create the Map index class.",
+                "",
+                "* Use index class type `AbstractIndexCreationTask`."
```

Comment:
same suggestion, for all places where we explaining why the index class should 'use' AbstractIndexCreationTask - it may be not obvious for users due to the class name

Reply:
done - in all places